pytest_splunk_addon/splunk.py

```diff
@@ -171,6 +172,13 @@ def pytest_addoption(parser):
         type=int,
         help="Time interval to wait before retrying the search query.",
     )
+    group.addoption(
+        "--cim-report",
+        action="store",
+        dest="cim_report",
+        help="create html report file at given path.",
```

Comment:
Need to change. `Create a markdown report summarizing  CIM compliance.`

Reply:
Done.